scripts/build/get-dir-size.js

```diff
@@ -0,0 +1,32 @@
+// @ts-check
+const { lstatSync, readdirSync } = require("fs");
+const { join } = require("path");
+const { promisify } = require("util");
+const execFile = promisify(require("child_process").execFile);

```

Comment:
I don't know if is needed, but promisify not exists in node 6.

Reply:
Also, `execFile` isn't used?

Reply:
Oops, this was a bad copypasta on my part 😟